repositories/decisions_repository.go

```diff
@@ -71,6 +71,29 @@ func (repo *MarbleDbRepository) DecisionWithRuleExecutionsById(ctx context.Conte
 	return decisions[0], nil
 }
 
+func (repo *MarbleDbRepository) DecisionsByOutcome(ctx context.Context, exec Executor,
+	scenarioID string,
+) ([]models.DecisionsByVersionByOutcoume, error) {
+	query := NewQueryBuilder().
+		Select("d.outcome, d.scenario_version, Count(d.outcome) as total").
+		From(fmt.Sprintf("%s AS d", dbmodels.TABLE_DECISIONS)).
+		Join(fmt.Sprintf("%s as sp on sp.scenario_iteration_id = d.scenario_iteration_id",
```

Comment:
why the join with scenario publications ?
I think we just need to join decisions and scenario_iterations, and filter by decision created_at